Dockerfile

```
@@ -1,32 +1,35 @@
 # syntax=docker/dockerfile:1.4
-FROM lukemathwalker/cargo-chef:0.1.51-rust-1.67-slim AS chef
-WORKDIR /app
-
-FROM chef AS planner
-COPY --link . .
-RUN cargo chef prepare --recipe-path recipe.json
+FROM rust:1.68-slim AS build-env
 
-FROM bufbuild/buf:1.15.1 as buf
+WORKDIR /app
 
-FROM namely/protoc:1.42_2 as protoc
+RUN --mount=target=. \
+    --mount=type=cache,target=/usr/local/cargo/git/db \
+    --mount=type=cache,target=/usr/local/cargo/registry/cache \
+    --mount=type=cache,target=/usr/local/cargo/registry/index \
+    cargo fetch --manifest-path app/server/Cargo.toml
```

すいません、スラッシュ抜けてました・・・

```suggestion
    cargo fetch --manifest-path /app/server/Cargo.toml
```